apps/frontend/src/app/(waitlist)/waitlist/page.tsx

```
@@ -0,0 +1,29 @@
+import React from "react"
+
+import Image from "next/image"
+
+import waitlist from "../../../../public/icons/logo.svg"
+import { LogoDark } from "@/src/lib/icons/Logo"
+
+type Props = {}
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)